packages/preview/arxivify/0.1.0/README.md

```diff
@@ -0,0 +1,53 @@
+# arxivify
+
+A Typst template based on popular LateX template used in NeurIPS, arXiv and bio-arXiv. Inspired by [arxiv-style](https://github.com/kourgeorge/arxiv-style)
+
+![Example](template/example.png)
```

Comment:
I would recommend not putting the thumbnail into the README (or at least not at the top) because the thumbnail will be shown on Typst Universe separately anyways.